src/client.cpp

```diff
@@ -289,6 +289,9 @@ extern "C" {
 		pCapabilities->bSupportsRecordingPlayCount	= true;
 		pCapabilities->bSupportsLastPlayedPosition	= g_bEnableMultiResume;
 		pCapabilities->bSupportsRecordingEdl		= false;
+		pCapabilities->bSupportsRecordingsRename	= true;
```

Comment:
Is this meant to be set true? Are any code changes needed to support recording rename capability?